sdk/python/tests/components/test_python_op.py

```diff
@@ -49,7 +49,7 @@ def helper_test_2_in_1_out_component_using_local_call(self, func, op):
                 task = op(arg1, arg2)
 
             full_command = task.command + task.arguments
-            process = subprocess.run(full_command)
+            subprocess.check_call(full_command)
```

Comment:
Why not set the "check" in subprocess.run call(https://docs.python.org/3/library/subprocess.html#subprocess.run).

subprocess,check_call is one of the old API(https://docs.python.org/3/library/subprocess.html#older-high-level-api), which i do not think should be used extensively.

Reply:
Thanks for the insight.
I've switched back to `subprocess.run`.